### DISCUSSION TITLE
LF to CRLF

### DISCUSSION THREAD

**Discussion by mumin16** [nlohmann/json#4071] [General]
json data;
....
stream s=data.dump();

is create CRLF stream. but if i use

json data;
....
stream s=data.dump(4);

is create LF stream. but i need CRLF stream(with dump(4)).

**Comment by nlohmann** [comment]
The library does not support this.

**Comment by gregmarr** [comment]
To clarify what you're seeing, `data.dump()` output does not contain newlines, but `data.dump(4)` does, is that accurate?
